extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -1062,35 +991,32 @@ export class AmpLightboxGallery extends AMP.BaseElement {
    * @param {!../../../src/service/action-impl.ActionInvocation} invocation
    * @private
    */
-  open_(invocation) {
-    let target = invocation.caller;
-    if (invocation.args && invocation.args['id']) {
-      const targetId = invocation.args['id'];
-      target = this.getAmpDoc().getElementById(targetId);
-      userAssert(target,
-          'amp-lightbox-gallery.open: element with id: %s not found', targetId);
-    }
-    this.openLightboxGallery_(dev().assertElement(target)).then(() => {
-      return this.history_.push(this.close_.bind(this));
-    }).then(historyId => {
-      this.historyId_ = historyId;
-    });
+  handleOpenAction_(invocation) {
+    const args = invocation.args || {};
```

Comment:
Nit for terseness. Also, personal preference nit: `handle` is redundant imo, just `openOnAction_`?
```js
openOnAction_({args = {}}) {
  const id = args['id'];
  // ...
```